clarifai/client/client.py

```diff
@@ -136,7 +136,7 @@ def __init__(self, app_id=None, app_secret=None, base_url='https://api.clarifai.
     self.api_info = None
 
   def set_model(self, model):
-    self._model = self._sanitize_param(model, 'default')
+    self._model = self._sanitize_param(model)
```

Comment:
If I read the current code right, we're already not sending an explicit model param in the default case.  self._model is None, and _sanitize_param returns None when it gets None.  I think this change isn't necessary.